catalogue/webapp/components/WorkDetails/WorkDetailsNewDataGrouping.js

```diff
@@ -15,7 +15,7 @@ import License from '@weco/common/views/components/License/License';
 import Divider from '@weco/common/views/components/Divider/Divider';
 import CopyUrl from '@weco/common/views/components/CopyUrl/CopyUrl';
 import Button from '@weco/common/views/components/Buttons/Button/Button';
-import MetaUnit from '@weco/common/views/components/MetaUnit/MetaUnit';
+import MetaUnit2 from '@weco/common/views/components/MetaUnit/MetaUnit2';
```

Comment:
I'd be tempted just to alias it as `MetaUnit`, then if/when it becomes the only one we have, it won't be weirdly named (and we can change the filename at that point if we want, too).